src/components/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,186 @@
+@import '../../styles/colors';
+
+.Accordion {
+  --zui-accordion-margin-top: 0.25rem;
+  --zui-accordion-group-border-radius: 0.5rem;
+
+  .AccordionItem:first-child {
+    --zui-accordion-margin-top: 0;
+  }
+
+  .AccordionHeader {
+    --zui-accordion-border-bottom: 0.063rem solid #{$purple-50};
+
+    &:hover {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-8};
+    }
+
+    &:active {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-50};
+    }
+  }
+
+  .AccordionTrigger {
+    --zui-accordion-item-padding: 0.5rem 0 0.438rem;
+    --zui-accordion-header-background-color: transparent;
+
+    &:hover {
+      --zui-accordion-header-background-color: #{$purple-2};
+      --zui-accordion-item-padding: 0.5rem 0px 0.438rem;
+    }
+
+    &[data-state='open'] {
+      --zui-accordion-header-background-color: #{$purple-2};
+      --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+
+      &:hover {
+        --zui-accordion-header-background-color: #{$purple-2};
+        --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+      }
+    }
+
+    &:active {
+      --zui-accordion-header-background-color: #{$purple-2};
+    }
+  }
+
+  &[data-contrast='high'] {
+    .AccordionHeader {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-6};
+
+      &[data-state='open'] {
+        --zui-accordion-border-bottom: none;
+      }
+
+      &:hover {
+        --zui-accordion-border-bottom: 0.063rem solid #{$purple-8};
+      }
+
+      &:active {
+        --zui-accordion-border-bottom: 0.063rem solid #{$purple-6};
+      }
+    }
+
+    .AccordionTrigger {
+      --zui-accordion-header-background-color: #{$purple-4};
+      --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+
+      &[data-state='open'] {
+        --zui-accordion-header-background-color: #{$purple-2};
+      }
+
+      &:hover {
+        --zui-accordion-header-background-color: #{$purple-2};
+      }
+    }
+  }
+}
+
+.Accordion {
+  width: 100%;
+  color: $grey-12;
+
+  button,
+  h3 {
+    all: unset;
+  }
+
+  .AccordionItem {
+    overflow: hidden;
+    margin-top: var(--zui-accordion-margin-top);
+
+    &:first-child {
+      border-top-left-radius: var(--zui-accordion-group-border-radius);
+      border-top-right-radius: var(--zui-accordion-group-border-radius);
+    }
+
+    &:last-child {
+      border-bottom-left-radius: var(--zui-accordion-group-border-radius);
+      border-bottom-right-radius: var(--zui-accordion-group-border-radius);
+    }
+
+    &[data-focus-on-key-press='true']:focus-within {
+      border-radius: 0.25rem;
+      outline-offset: 0.25rem;
+      outline: 0.0625rem solid $green-11;
+    }
+  }
+
+  .AccordionHeader {
+    display: flex;
+    width: 100%;
+    border-bottom: var(--zui-accordion-border-bottom);
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  .AccordionTrigger {
```

Comment:
You can remove `Accordion` from all of these child classes - CSS modules will scope them correctly by default so you can use simple names.

Reply:
Updated